docs-src/concepts/what-is-the-temporal-web-ui.md

```diff
@@ -104,8 +106,22 @@ It works only if a Worker is running and available to return the stack trace.
 
 Lists all Queries sent to the Workflow Execution.
 
+### Schedules
+
+On Temporal Cloud and slef-hosted Temporal Cluster Web UI, the Schedules page lists all the [Schedules](/workflows#schedule) created on the selected Namespace.
+
+Click on a Schedule to see details, such as configured frequency, start and end time, recent and upcoming runs.
+
 ### Settings
 
+On Temporal Cloud, the Settings option shows only to [Global admins](/cloud/#account-level-roles).
```

Comment:
not sure what "option" means. how does this show up in the UI? an icon on the left? @Alex-Tideman

Reply:
It is this one:
<img width="566" alt="image" src="https://github.com/temporalio/documentation/assets/48060050/1086bb56-0245-4923-9804-8fd0b8d65ef9">